tests/foreman/cli/test_variables.py

```diff
@@ -283,7 +288,8 @@ def test_positive_update_variable_puppet_class(self):
         smart_variable = make_smart_variable({
             'puppet-class': self.puppet['name']})
         self.assertEqual(smart_variable['puppet-class'], self.puppet['name'])
-        new_puppet = Puppet.info({u'name': 'ntp::config'})
+        new_puppet = Puppet.info(
+            {u'name': self.puppet_modules[1]['name']})
```

Comment:
But after another thought I understand that indeed it can be a problem. Moving to WIP.